reference/docs-conceptual/PowerShell-Core-Support.md

```diff
@@ -0,0 +1,97 @@
+# PowerShell Core Support Lifecycle
+
+PowerShell Core is a distinct set of tools and components that is shipped, installed, and configured separately from Windows PowerShell.
+Therefore, PowerShell Core is not included in the Windows 7/8.1/10 or Windows Server licensing agreements.
+
+However, PowerShell Core is supported under traditional Microsoft support agreements, including [Premier][], [Microsoft Enterprise Agreements][enterprise-agreement], and [Microsoft Software Assurance][assurance].
+You can also pay for [assisted support][] for PowerShell Core by filing a support request for your problem.
+
+We also offer [community support][] on GitHub where you can file an issue, bug, or feature request.
+Alternatively, you may find help from other members of the community on the general [Microsoft Community][] or the Microsoft [PowerShell Tech Community][].
+We offer no guarantee there that your issue will be addressed or resolved in a timely manner.
+If you have a problem that requires immediate attention,
+you should use the traditional, paid support options.
+
+## Lifecycle of PowerShell Core
+
+PowerShell Core is adopting the [Microsoft Modern Lifecycle Policy][modern].
+This support lifecycle is intended to keep customers up-to-date with the latest versions.
+
+The version 6.x branch of PowerShell Core will be updated approximately once every six months (e.g. 6.0, 6.1, 6.2, etc.)
+
+> [!IMPORTANT]
+> You must update within six months after each new minor version release to continue receiving support.
```

Comment:
Any requirement to pickup revisions. 6.0.x for example?

Reply:
Yes, there is. We need to get this one merged, I'm going to take comments into a later PR so we can get this merged.